Docker/create-collection.py

```diff
@@ -0,0 +1,24 @@
+import http.client
+import json
+
+payload = ""
+
+conn = http.client.HTTPConnection("solr:8983")
+
+conn.request("GET", "/solr/admin/collections?action=LIST&wt=json", payload)
+
+res = conn.getresponse()
+data = res.read()
+parsed_json = (json.loads(data))
+
+# Check if collection is already present
+if 'mentions' not in parsed_json['collections']:
+    # Create collection if not present
+    conn.request("GET", "/solr/admin/collections?action=CREATE&name=mentions&numShards=1&replicationFactor=1", payload)
```

Comment:
Interesting that collections are created with a `GET` request and not `POST`

Reply:
Indeed. Didn't realized at the time writing but yes indeed.